docs/source/usage/01_usage.md

```diff
@@ -67,6 +67,7 @@ are shared among multiple roles:
 - `cifmw_openshift_ingress_ip_address` (String) contains the OpenShift Ingress IP address. Note: it is computed internally and should not be user defined.
 - `cifmw_nolog`: (Bool) Toggle `no_log` value for selected tasks. Defaults to `true` (hiding those logs by default).
 - `cifmw_parent_scenario`: (String or List(String)) path to existing scenario/parameter file to inherit from.
+- `cifmw_configure_switches`: (Bool) Specifies whether switches should be configured. Defaults to `false`.
```

Comment:
if we document that one, we must also mention the way it's computed in the reproducer.yml play imho.